backend/src/v5/middleware/dataConverter/inputs/teamspaces/projects/models/commons/revisions.js

```diff
@@ -53,36 +54,37 @@ Revisions.validateUpdateRevisionData = async (req, res, next) => {
 };
 
 const fileFilter = async (req, file, cb) => {
-	try {
-		const { originalname, size } = file;
-		const { teamspace } = req.params;
-		const fileExt = Path.extname(originalname);
-		if (!ACCEPTED_MODEL_EXT.includes(fileExt)) {
-			const err = createResponseCode(templates.unsupportedFileFormat, `${fileExt} is not a supported model format`);
-			cb(err, false);
-		} else {
-			await sufficientQuota(teamspace, size);
-			cb(null, true);
-		}
-	} catch (err) {
+	const { originalname } = file;
+	const fileExt = Path.extname(originalname).toLowerCase();
+	if (!ACCEPTED_MODEL_EXT.includes(fileExt)) {
+		const err = createResponseCode(templates.unsupportedFileFormat, `${fileExt} is not a supported model format`);
 		cb(err, false);
+	} else {
+		cb(null, true);
 	}
 };
 
 const validateRevisionUpload = async (req, res, next) => {
 	const schema = Yup.object().noUnknown().required()
 		.shape({
 			tag: YupHelper.types.strings.code.required(),
-			desc: YupHelper.types.strings.blob,
+			desc: YupHelper.types.strings.shortDescription,
 			importAnimations: Yup.bool().default(true),
 		});
 
 	try {
 		req.body = await schema.validate(req.body);
 		if (!req.file) throw createResponseCode(templates.invalidArguments, 'A file must be provided');
+
+		const { teamspace } = req.params;
+		await sufficientQuota(teamspace, req.file.size);
+
+		const isTagValid = await isValidTag(req.params.teamspace, req.params.container, req.body.tag);
```

Comment:
Can we do that as part of the test in L70?

I.e. line 70 should look something like this:

````js
  tag: YupHelper.types.strings.code.required().test("tag-not-in-use", "Revision name is already used by an existing revision", async () => { /* the tag test here*/})
````